x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/analytics/AnalyticsConstants.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.application.analytics;
+
+public class AnalyticsConstants {
```

Comment:
I will probably refactor this in the near future to create an `AnalytisEventDataStream` class with more utils methods.
I propose to keep it for now and remove it  later since these constants does not clearly belong to one of the existing class.